docs/getting-started/ios/screenshots.md

```diff
@@ -126,7 +126,7 @@ languages([
 # fastlane action snapshot
 ```
 
-You can adapt this file to fit your project. Every time you run `fastlane snapshot` the file will be loaded automatically.
+You can adapt this file to fit your project (alternatively you could add your snapshot process to a lane in your fastfile). Every time you run `fastlane snapshot` the file will be loaded automatically.
```

Comment:
Let's remove this, since this section is all about the Snapfile, and not the Fastfile